From de42705b3fe1491154265754f8289d5c2fb03c44 Mon Sep 17 00:00:00 2001
From: dianlujitao <dianlujitao@lineageos.org>
Date: Thu, 10 Jan 2019 12:12:37 +0800
Subject: [PATCH 2/9] Revert "common: Add sf_lcd_density_prop type and labelled
 props"

 * Not work, and not needed, the property is labelled by AOSP sepolicy
   on Pie.

This reverts commit d54feea5d968f51d66421605e178be1f43015dd5.

Change-Id: I1c746d6c5cd24f8c3161973f11a2d8f6c871b246
---
 common/private/recovery.te      | 1 -
 common/public/property.te       | 1 -
 common/public/property_contexts | 1 -
 3 files changed, 3 deletions(-)
 delete mode 100644 common/public/property.te
 delete mode 100644 common/public/property_contexts

diff --git a/common/private/recovery.te b/common/private/recovery.te
index 1393a1f..e788026 100644
--- a/common/private/recovery.te
+++ b/common/private/recovery.te
@@ -2,7 +2,6 @@ recovery_only(`
 
 # XXX
 ##allow init rootfs:file write;
-get_prop(recovery, sf_lcd_density_prop)
 r_dir_file(recovery, pstorefs)
 allow recovery system_file:dir mounton;
 
diff --git a/common/public/property.te b/common/public/property.te
deleted file mode 100644
index 81735e0..0000000
--- a/common/public/property.te
+++ /dev/null
@@ -1 +0,0 @@
-type sf_lcd_density_prop, property_type;
diff --git a/common/public/property_contexts b/common/public/property_contexts
deleted file mode 100644
index 36f5f3b..0000000
--- a/common/public/property_contexts
+++ /dev/null
@@ -1 +0,0 @@
-ro.sf.lcd_density             u:object_r:sf_lcd_density_prop:s0
-- 
2.17.1

